proposal submission static page (#637)
urbit/urbit.org: content/grants/proposals/submit.md

+++
title = "Guide to Creating a Grant Proposal"
weight = 1
description = "A comprehensive guide to creating a grant proposal."
+++

> The Urbit project relies on our community to help realize its potential. While the Foundation specifys the direction of 

Reviewer: ```suggestion
The Urbit project relies on our community to help realize its potential. While the Foundation specifies the direction of 
```

development through bounties, we encourage anyone with an idea about how to improve Urbit to submit a grant proposal. This
guide will describe the different types of proposals, the requested format, and the entire workflow for a successful project.

### Types of Proposals

While we will entertain just about any idea that comes from the community, we are particularly interested in "capital goods":

> projects that extend usability or add tooling to the platform. Recent examples are 

Reviewer: "extend usability" is vague. Let's say something to the effect of "projects that provide new capabilities to the widest number of users or developers"

> [Bitcoin wallet](../bitcoin-full-node-provider-and-wallet) integration, [WebRTC](../webrtc-gall-agent-and-external-app)

Reviewer: ```suggestion
[Bitcoin](../bitcoin-full-node-provider-and-wallet) integration, [WebRTC](../webrtc-gall-agent-and-external-app)
```

> implementation, or the [Urbit Large File Sytem](../urbit-lfs-filehosting) project.   

Reviewer: ```suggestion
integration, or the [Urbit Large File Storage](../urbit-lfs-filehosting) project.   
```

Proposal types include:

>  - Core Development - low level network or kernaldevelopment

Reviewer: ```suggestion
 - Core Development - kernel or runtime development
```

 - Application Development - userspace development of specific applications
 - Developer Tools - Libraries, APIs, unit tests, etc.

>  - Design Work - overall website design work

Reviewer: ```suggestion
 - Design Work - design work for the urbit.org website
```

>  - Documentation - writing guides and website content

Reviewer: ```suggestion
 - Documentation - writing guides or website content
```

 - Community - organizing events and meetups, staffing the help desk

> We have need for lots of different skill sets - not just technical.

Reviewer: ```suggestion
We have need for lots of different skill sets&mdash;not just coding.
```

### Proposal Format

Your proposal should include the following sections:

 - Project name 
 - Brief description of project
 - Proposal body including:
   - Project Overview
   - Project Scope
   - Detailed behavior description
   - Detailed design description
   - Your experience
   - Why you are the right person for this project
 - Milestones:
   - Any sub-goals of project
   - Specific deliverables
   - Expected timeframes
   - Expected compensation in Stars

### Proposal Submission and Project Workflow

Once you have your proposal in order, submit it using the **[Propose a Project](https://airtable.com/shrCi54rEDxgSZr3z)** form. 
From this point, your project will go through these stages:

>  - **Draft**: You will schedule a meeting with a team member to discuss your proposal in detail. There may be some modifications 

Reviewer: It may not be clear how one will schedule a meeting with a team member. Probably better to say that we'll reach out to schedule a meeting, which in effect is just the calendly link.

requested to get approval. Once we come to agreement you will create markdown for your project by 

> [downloading the template](https://drive.google.com/file/d/1a2zZXxmzcME5vJHDpLpKtExcqzg-V3rD/view?usp=sharing), editing it with 

Reviewer: This template is great. Nice work!
